common/laws/00_church_and_state.txt

```diff
@@ -306,7 +317,17 @@ law_total_separation = {
 				ai_has_enact_weight_modifier_journal_entries = yes
 			}
 			add = 750
-		}			
+		}
+		if = {
+			limit = { 
+				OR = {
+					has_law = law_type:law_state_religion
+					has_law = law_type:law_freedom_of_conscience
+				}
+				has_journal_entry = je_sick_man_liberal_secularism
+			}
+			add = 1000
+		}
```

Comment:
The journal entry needs to reflect that. Currently atheism is not valid for completion of the journal entry.

Reply:
I think you're better of defining script values, where you use the same script value for the completion of the journal entry, and the NOT = {script_value} for the ai law prioritization.

Reply:
I'll get to it after this patch. Just a lot to do right now for a clerical change

Reply:
What does atheism have to do with total separation?

Reply:
Because in the hypothetical case that the ottoman has atheism, it won't try ans get seperation, eventhough they need it for the journal entry.